docs/commands/process.md

```diff
@@ -0,0 +1,59 @@
+# `process` command
+
+The `process` command extends the `gopass` templating to support user-supplied
+template files that will be processed and any necessary credential lookups will
+be done while they are processed. That way users can store their full
+configuration files publicly accessible and have any of the recipients
+automatically populate it to generate a complete configuration file.
```

Comment:
This really isn't super clear to me what do you mean with "any necessary credential lookups will be done"? 
Also it isn't super clear that the usecase is to generate configuration files that contains secrets on the fly. (Or is it just meant to generate them once and store them?)

Reply:
Clarified. Thank you.